data/cube-nurbs.mesh

```diff
@@ -0,0 +1,73 @@
+MFEM NURBS mesh v1.0
```

Comment:
This same file is duplicated in `miniapps/nurbs`

Reply:
Removed